guides/common/modules/con_smartproxy_performance_tests.adoc

```diff
@@ -24,10 +24,17 @@ For more precise numbers, see the table below.
 When we compared download performance from {ProjectServer} vs. from {SmartProxyServer}, we have seen only about 5% speedup, but that is expected as {SmartProxyServer}'s main benefit is in getting content closer to geographically distributed clients (or clients in different networks) and in handling part of the load {ProjectServer} would have to handle itself.
 In some smaller hardware configurations (8 CPUs and 24 GiB), {ProjectServer} was not able to handle downloads from more than 500 concurrent clients, while a {SmartProxyServer} with the same hardware configuration was able to service more than 1000 and possibly even more.
 
-.Frequent registrations use case
-For concurrent registrations a bottleneck is CPU speed, but all configs were able to handle even high concurrency without swapping.
+.Concurrent registrations use case
+For concurrent registrations a bottleneck is usually CPU speed, but all configs were able to handle even high concurrency without swapping.
 Hardware resources used for {SmartProxy} have only minimal impact on registration performance.
 For example, {SmartProxyServer} with 16 CPUs and 46 GiB RAM have at most a 9% registration speed improvement when compared to a {SmartProxyServer} with 4 CPUs and 12 GiB RAM.
+During periods of very high concurrency, you might experience timeouts in the {SmartProxyServer} to {ProjectServer} communication.
+You can alleviate this by increasing the default timeout using the following tunable in "/etc/foreman-installer/custom-hiera.yaml":
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+apache::mod::proxy::proxy_timeout: 600
```

Comment:
As I've stated before: I don't want to document `/etc/foreman-installer/custom-hiera.yaml`. It would imply that it's a supported solution, but it's not. It's a workaround.

If we recommend this to users, we should make a properly supported feature in the installer. Is there an issue somewhere tracking that effort?

Reply:
Hi Ewoud, I have to disagree here. We have a KCS addressing this very issue (https://access.redhat.com/solutions/3412321), which means that the solution is supported, wether we like it or not.

Reply:
@pablomh Do we have a BZ tracking first class support for adding this Apache timeout?

Reply:
Hi Eric. Do you mean like proposing the creation of something like `--foreman-apache-proxy-timeout` as installer option?

Reply:
New BZ opened for this issue: [[RFE] Add parameter to installer to set capsule's `proxy_timeout`](https://bugzilla.redhat.com/show_bug.cgi?id=2189587)